src/CMake/cpack.cmake

```diff
@@ -36,6 +36,11 @@ if (${LINUX_FLAVOR} MATCHES "^(Ubuntu|Debian)")
   SET(CPACK_DEBIAN_AWS_PACKAGE_DEPENDS "xrt (>= ${XRT_VERSION_MAJOR}.${XRT_VERSION_MINOR}.${XRT_VERSION_PATCH})")
   # xrt component dependencies
   SET(CPACK_DEBIAN_XRT_PACKAGE_DEPENDS "ocl-icd-opencl-dev (>= 2.2.0), libboost-dev (>=1.58), libboost-filesystem-dev (>=1.58), uuid-dev (>= 2.27.1), dkms (>= 2.2.0), libprotoc-dev (>=2.6.1), protobuf-compiler (>=2.6.1), libncurses5-dev (>=6.0), lsb-release, libxml2-dev (>=2.9.1), libyaml-dev (>= 0.1.6)")
+  if (${LINUX_VERSION} STREQUAL 16.04)
+    SET(CPACK_DEBIAN_XRT_PACKAGE_DEPENDS "${CPACK_DEBIAN_XRT_PACKAGE_DEPENDS}, libc6 (>= 2.23), libc6 (< 2.27)")
+  else ()
```

Comment:
Add an explicit check for 18.04 version as well

Reply:
What should we do in the else condition where version isn't 18.04 or 16.04?

Reply:
Maybe we can get the glibc version dynamically?
Can we try something like this with proper credits--
https://gist.github.com/likema/f5c04dad837d2f5068ae7a8860c180e7

Reply:
Try on xsjsycl40 and you will know... :-) Otherwise, add 18.10 and 19.04, since they are still valid versions. Probably that picking the highest LTS will just work because the package versions on the target will have higher versions anyway...

Reply:
I'm not sure how to use that glibc macro and ensure that the build version matches the install version.